apps/web/pages/[user].tsx

```diff
@@ -44,6 +45,10 @@ export function UserPage(props: InferGetServerSidePropsType<typeof getServerSide
   useTheme(profile.theme);
   const { t } = useLocale();
 
+  const linkedinLink = (user?.metadata as Prisma.JsonObject)?.linkedinLink as string;
+  const Xlink = (user?.metadata as Prisma.JsonObject)?.XLink as string;
```

Comment:
What do you think about the following constant name casing to respect camel case:
1. It's LinkedIn (i in In is capital), so need to have constant `const linkedInLink`
2. `camelCase` requires that first letter is small, what about having `xLink` instead of `Xlink?

Reply:
Also, why is it necessary to type cast? Can't we:
1. In `isBioEmpty` we check `!user.bio` -> we don't write `!user?.bio` - can we write `user.metadata` ?
2. Can we omit casting it as a string and allowing LinkedIn link or X Link to be undefined -> user.metadata?.xLink

and then showSocial boolean would be truthy or falsy.

Reply:
` user.metadata?.xLink` using this gives me an error 
Property 'linkedIn' does not exist on type 'string | number | boolean | JsonObject | JsonArray'.